src/test/javascript/portal/details/LayerControlPanelSpec.js

```diff
@@ -70,7 +70,7 @@ describe("Portal.details.LayerControlPanel", function() {
 
             layerSelectComponent.fireEvent('change', layerSelectComponent, layerSelectComponent.items[0]);
             expect(window.trackLayerControlUsage).toHaveBeenCalledWith(
-                'changeLayerTrackingAction',
+                'change layer',
                 'Data Collection Title',
```

Comment:
Isn't this in the wrong order?

Reply:
`action` then `label`
The action is now a string not i18n

Reply:
Yes I can see that, but the parameters expected for trackLayerControlUsage are now action, label, collection - these parameters don't match that.  See https://github.com/aodn/aodn-portal/blob/26a62c3a0379398908fc6d01cdf29871e16e6cf7/web-app/js/portal/utils/trackUsage.js#L69

Reply:
Ok so i now can see on the GA page how to see the collection dimension
Back to the drawing board